assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -656,3 +656,9 @@ che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/nightly
 
 # Docker image for the Che async storage
 che.infra.kubernetes.async.storage.image=quay.io/eclipse/che-workspace-data-sync-storage:latest
+
+# The timeout for the Asynchronous Storage Pod shutdown after stopping the last used workspace.
+che.infra.kubernetes.async.storage.shutdown_timeout_min=120
+
+# Defines the period with which the Asynchronous Storage Pod stopping ability will be performed (once in 120 minutes by default)
```

Comment:
sorry, here should be 30 minutes, not 120 according to the property value

Reply:
np, i will fix